### DISCUSSION TITLE
Color Coding Variables

### DISCUSSION THREAD

**Discussion by FlamedDogo99** [ScratchAddons/ScratchAddons#6370] [Ideas]
When working with many variables, it can be hard to quickly pick out the desired item. Because of this, I propose adding a small optional color indicator on each variable itself before the text. This would be set through the variable editing screen, and be saved in the project itself, similar to how turbowarp can package settings into a project. It would also display on variable dropdowns.

**Comment by WorldLanguages** [comment]
As a workaround, you can use emojis in the variable name.

**Reply by FlamedDogo99** [reply]
Actually that’s really helpful. I still think it would be a nice quality of life feature but I’ll definitely start using this.
